pkg/detectors/azurestorage/azurestorage.go

```diff
@@ -44,47 +51,15 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 
 		s1 := detectors.Result{
 			DetectorType: detectorspb.DetectorType_AzureStorage,
-			Raw:          []byte(accountName),
+			Raw:          []byte(accountKey),
```

Comment:
this is the main fix for bug

Reply:
is the account name sensitive information? if not, i think we should add it to `ExtraData` (like we do for aws keys). that could be helpful in support cases and just generally useful to users.

Reply:
no it's not sensitive, that's a good idea